src/components/DataGridWithInfiniteScroll/DatagridInfiniteScrollMockData.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export const DATA = {
```

Comment:
```suggestion
export const rowData= {
```

Please use consistent style for variable/constant names

Reply:
Done